### PR TITLE
Add tecOVERSIZE transaction result

### PR DESCRIPTION
**Pull request by wltsmrz**

### CODE REVIEW

File: src/core/binformat.js [hunk 1]
```diff
@@ -423,7 +423,7 @@ exports.ter = {
   tecNO_LINE_REDUNDANT     : 127,
   tecPATH_DRY              : 128,
   tecUNFUNDED              : 129,  // Deprecated, old ambiguous unfunded.
-  tecMASTER_DISABLED       : 130,
+  tecNO_ALTERNATIVE_KEY    : 130,
```
Comment (sublimator) [inline src/core/binformat.js]: Hrmm, this was changed huh ?

Comment (sublimator) [inline src/core/binformat.js]: A little bit `hrmmmmmmmm` cause now old json won't serialise properly. When did rippled make this change?

Comment (shekenahglory) [inline src/core/binformat.js]: it wont fail right? just the output will be tecNO_ALTERNATIVE_KEY when it used to say tecMASTER_DISABLED?

**Comment by wltsmrz** [comment]
Let's wait until https://github.com/ripple/ripple-lib/pull/545

File: src/core/binformat.js (hunk 1 above)
Comment (sublimator) [inline src/core/binformat.js]: Talking about old json -> binary.

**Comment by sublimator** [comment]
LGTM

**Pull request merged**